src/cli/onefuzz/api.py

```diff
@@ -1290,70 +1290,164 @@ def config(
             answer = input(" ".join(message)).strip()
 
         if answer == "n":
             self.logger.info("not stopping")
-            return
-
-        self.logger.info("stopping all interactions")
+            return False
+        return True
 
+    def _delete_jobs(self):
         for job in self.jobs.list():
             self.logger.info("stopping job %s", job.job_id)
             self.jobs.delete(job.job_id)
 
+    def _delete_tasks(self):
         for task in self.tasks.list():
             self.logger.info("stopping task %s", task.task_id)
             self.tasks.delete(task.task_id)
 
+    def _delete_notifications(self):
         for notification in self.notifications.list():
             self.logger.info("stopping notification %s", notification.notification_id)
             self.notifications.delete(notification.notification_id)
 
+    def _delete_repros(self):
         for vm in self.repro.list():
             self.repro.delete(str(vm.vm_id))
 
+    def _delete_pools(self):
+        for pool in self.pools.list():
+            self.logger.info("stopping pool: %s", pool.name)
+            self.pools.shutdown(pool.name, now=True)
+
+    def _delete_scalesets(self):
+        for scaleset in self.scalesets.list():
+            self.logger.info("stopping scaleset: %s", scaleset.scaleset_id)
+            self.scalesets.shutdown(scaleset.scaleset_id, now=True)
+
+    def _delete_containers(self):
+        for container in self.containers.list():
+            if (
+                container.metadata
+                and "container_type" in container.metadata
+                and container.metadata["container_type"]
+                in [
+                    "analysis",
+                    "coverage",
+                    "crashes",
+                    "inputs",
+                    "no_repro",
+                    "readonly_inputs",
+                    "reports",
+                    "setup",
+                    "unique_reports",
+                ]
+            ):
+                self.logger.info("removing container: %s", container.name)
+                self.containers.delete(container.name)
+
+    def _delete_all_running_jobs(self):
+        self._delete_jobs()
+        self._delete_tasks()
+        self._delete_notifications()
+        self._delete_repros()
+
+    def _delete_fuzzing_data(self):
+        self._delete_scalesets()
+        self._delete_pools()
+        self._delete_containers()
+
+    def reset(
+        self,
+        *,
+        containers: bool = False,
+        everything: bool = False,
+        jobs: bool = False,
+        notifications: bool = False,
+        pools: bool = False,
+        repros: bool = False,
+        scalesets: bool = False,
+        tasks: bool = False,
+        yes: bool = False,
+    ) -> None:
+        """
+        Resets onefuzz. Stops all jobs, notifications, and repro VMs.
+        Specifying 'everything' will delete all containers, pools, and managed
```

Comment:
@bmc-msft I don't see the help in cli from this line. Only first line is parsed and printed. Is there any particular reason for it?